src/components/views/globals/MatrixToolbar.js

```diff
@@ -32,6 +37,7 @@ module.exports = React.createClass({
     },
 
     render: function() {
+        if (this.state.showToolbar != true) return null;
```

Comment:
Cool, though if the user clicked on block in the notification permission dialog, we are going to show `you have disabled desktop notification` right? The logic for that would come here right?